miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/LibraryRestController.java

```diff
@@ -123,15 +125,15 @@ public LibraryDto getLibraryById(@PathVariable Long libraryId) throws IOExceptio
     return dto;
   }
 
-  @RequestMapping(method = RequestMethod.GET, produces = "application/json")
+  @GetMapping(produces = "application/json")
   @ResponseBody
   public List<LibraryDto> listAllLibraries() throws IOException {
     Collection<Library> libraries = libraryService.list();
     List<LibraryDto> dtos = Dtos.asLibraryDtos(libraries);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "dtos". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)